docs/en_US/dependency_management.md

```diff
@@ -237,17 +278,66 @@ GetMaterialApp(
-The "full" mode is also safe enough to be used with StatelessWidget, as it has numerous security callbacks that will prevent a controller from remaining in memory if it is not being used by any widget, and disposition is not important here. However, if you are bothered by the default behavior, or just don't want it to happen, Get offers other, more lenient options for intelligent memory management, such as SmartManagement.onlyBuilders, which will depend on the effective removal of widgets using the controller. tree to remove it, and you can prevent a controller from being deployed using "autoRemove: false" in your GetBuilder/GetX.
-With this option, only controllers started in "init:" or loaded into a Binding with "Get.lazyPut" will be disposed, if you use Get.put or any other approach, SmartManagement will not have permissions to exclude this dependency.
 With the default behavior, even widgets instantiated with "Get.put" will be removed, unlike SmartManagement.onlyBuilders.
-SmartManagement.keepFactory is like SmartManagement.full, with one difference. SmartManagement.full purges the factories from the premises, so that Get.lazyPut() will only be able to be called once and your factory and references will be self-destructing. SmartManagement.keepFactory will remove its dependencies when necessary, however, it will keep the "shape" of these, to make an equal one if you need an instance of that again.
+
+#### SmartManagement.keepFactory
+
+// TODO ASK JONATAS WHAT THE HECK HE MEANT WITH THIS TEXT
+This one is like SmartManagement.full, with one difference: SmartManagement.full purges the factories from the premises, so that Get.lazyPut() will only be able to be called once and your factory and references will be self-destructing. SmartManagement.keepFactory will remove its dependencies when necessary, however, it will keep the "shape" of these, to make an equal one if you need an instance of that again.
+
 Instead of using SmartManagement.keepFactory you can use Bindings.
+// TODO ASK JONATAS THIS TOO
 Bindings creates transitory factories, which are created the moment you click to go to another screen, and will be destroyed as soon as the screen-changing animation happens. It is so little time that the analyzer will not even be able to register it. When you navigate to this screen again, a new temporary factory will be called, so this is preferable to using SmartManagement.keepFactory, but if you don't want to create Bindings, or want to keep all your dependencies on the same Binding, it will certainly help you . Factories take up little memory, they don't hold instances, but a function with the "shape" of that class you want. This is very little, but since the purpose of this lib is to get the maximum performance possible using the minimum resources, Get removes even the factories by default. Use whichever is most convenient for you.
```

Comment:
This part too, if i be able to understand, i can refactor this to be more beginner friendly.

Also, this is explaining how GetX works under the hood, but there is another part for this already, is this necessary? 🤔 
It may be because is explaining more Bindings, but i am in doubt

Reply:
Basicamente isso explica como Bindings funciona, no momento que a rota é chamada, ele é executado, e as "factorys" do lazyPut são destruídas assim que ele for chamado (geralmente durante o build), e por contato, com o modo full, você pode usar Bindings sem problemas, mas o keepFactory você não pode usar mais de um Bindings, porque senão você vai criar uma "fabrica" toda vez q entrar na rota, e elas não vão ser destruídas.